extensions/BMO/Extension.pm

```diff
@@ -89,6 +89,50 @@ BEGIN {
     *Bugzilla::Attachment::fetch_github_pr_diff     = \&_attachment_fetch_github_pr_diff;
 }
 
+sub template_before_process_wants {
+    return {
+        'bug/create/create-user-engagement.html.tmpl'        => 1,
```

Comment:
In ```BugModal/Extension.pm``` this is done more cleanly with ```want{$_} = 1 for (```, which seems useful here as well. return(map { $_ => 1 } ...) or similar would let you format this list exactly as the one in BugModal, with only the entry/exit lines varying for context.